packages/normalizr/src/entities/__tests__/Entity.test.ts

```diff
@@ -233,17 +233,16 @@ describe(`${Entity.name} normalization`, () => {
   });
 });
 
-/* TODO: This isn't supported quite yet
```

Comment:
note we re-enable this entire block. This has a lot of interested recursively nested denormalization cases.